compiler/optimizer/Inliner.cpp

```diff
@@ -2382,6 +2382,63 @@ TR_ParameterToArgumentMapper::lookForModifiedParameters(TR::Node * node)
       }
    }
 
+void
+TR_ParameterToArgumentMapper::mapOSRCallSiteRematTable(uint32_t siteIndex)
```

Comment:
Does this need to check for OSR infrastructure being removed as part of late inlining?